admin-ui/src/pages/settings-page.tsx

```
@@ -137,6 +143,17 @@ function reasoningItemsFromRecord(
   }))
 }
 
+function aliasItemsFromRecord(
+  record: Record<string, string> | undefined
```

ESLint

[error] Insert `,`

(prettier/prettier)